packages/core/docs/recipes/redis.md

```diff
@@ -0,0 +1,101 @@
+# Use of redis{Cluster|Sentinel} plugins
```

Comment:
I would rather prefer the whole word usage for the search optimization purposes :)